connectomics/data/utils/data_io.py

```diff
@@ -49,7 +54,7 @@ def readimgs(filename):
 
     return data
 
-def writeh5(filename, dtarray, dataset='main'):
+def writeh5(filename, filepath, dtarray, dataset='main', saveAsPng = False):
```

Comment:
Here, we need a filepath to save the PNG images. Either you could pass the filepath separately or use the filename to strip the H5 filename. I have done the former. Open to any other design changes!